flags.go

```diff
@@ -209,6 +209,21 @@ func newApp() (app *cli.App) {
 					"inodes.",
 			},
 
+			cli.DurationFlag{
+				Name:  "Go-Storage-timeout",
```

Comment:
see the format of other flags - all letters should start with lower case

Reply:
fixed